app/controllers/katello/api/v2/repository_sets_controller.rb

```diff
@@ -14,7 +14,7 @@ class Api::V2::RepositorySetsController < Api::V2::ApiController
     end
 
     api :GET, "/products/:product_id/repository_sets", N_("List repository sets for a product.")
```

Comment:
`product_id` is required. I do believe another possibility would be to simply provide organization, although I have not tested this theory and believe it's out of the scope of this particular BZ.

Reply:
Good catch, @jturel! I tested this index endpoint without product_id, and it works great. We should add another `api` for `organizations/:organization_id/repository_sets`, but maybe not in this issue.

@cfouant I think for this issue, you can revert the `product_id` `required: false` change and be done with it.

Reply:
If that's the case why was required changed to false on line 17 -> `param :product_id, :number, :required => false` 

If it's valid it would be helpful to capture the reason here.

Reply:
I think you don't get a 404 cause the `api` is just a documentation method - the routes.rb still defines the actual routes. I was unclear on that myself the first time I looked at our API stuff